apps/frontend/src/views/Compare.vue

```diff
@@ -260,6 +267,17 @@ export default class Compare extends Vue {
     }
   ];
 
+  compareItems = [
+    'Run Time',
```

Comment:
`Run Time` is not the ideal word to use here, `runtime` normally means how long something takes to run, this is comparing the actual start time. This should probably be called `Scan Start Time` or maybe just `Start Time`